__tests__/app.test.js

```diff
@@ -57,8 +57,50 @@ describe('App', () => {
       .expect(200)
       .then((response) => {
         const body = response.body
-        expect(body).toEqual(endpoints);
+        expect(body.endpoints).toEqual(endpoints);
       })
     });
   });
+
+  describe('GET /api/articles/:article_id', () => {
+    test('should return a status code of 200 and the correct article', () => {
+      return request(app)
+      .get('/api/articles/3')
+      .expect(200)
+      .then((response) => {
+        const body = response.body
+        expect(body.article).toEqual({
```

Comment:
A flexible matcher might be better to use here in case the endpoint gets more data later